operators/registry-operator/0.1.1/manifests/registry-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,860 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "registry.devfile.io/v1alpha1",
+          "kind": "DevfileRegistry",
+          "metadata": {
+            "name": "sample-devfileregistry"
+          },
+          "spec": {
+            "devfileIndex": {
+              "image": "quay.io/devfile/devfile-index:next"
+            }
+          }
+        },
+        {
+          "apiVersion": "registry.devfile.io/v1alpha1",
+          "kind": "DevfileRegistriesList",
+          "metadata": {
+            "name": "sample-devfileregistrieslist"
+          },
+          "spec": {
+            "devfileRegistries": [
+              {
+                "name": "devfile-staging",
+                "url": "https://registry.stage.devfile.io"
+              }
+            ]
+          }
+        },
+        {
+          "apiVersion": "registry.devfile.io/v1alpha1",
+          "kind": "ClusterDevfileRegistriesList",
+          "metadata": {
+            "name": "sample-clusterdevfileregistrieslist"
+          },
+          "spec": {
+            "devfileRegistries": [
+              {
+                "name": "devfile-staging",
+                "url": "https://registry.stage.devfile.io"
+              }
+            ]
+          }
+        }
+      ]
+    capabilities: Basic Install
+    catagories: OpenShift Optional,Developer Tools
```

Comment:
https://github.com/k8s-operatorhub/community-operators/actions/runs/6305397546/job/17118786267#step:6:1350

This is claiming the `metadata.annotations.catagories` is missing however this is an existing field.

Reply:
Typo: should be categories

Reply:
@ncdc 

Spell check and I missed this detail :facepalm: 

Thanks for your feedback!

Reply:
fixed in https://github.com/k8s-operatorhub/community-operators/pull/3287/commits/94851330b77a47736e6b61071e30b6a50f950bd9